sdscli/adapters/hysds/status.py

```diff
@@ -79,7 +79,7 @@ def print_es_status(host):
     service = "elasticsearch"
     status = "inactive"
     try:
-        es = elasticsearch.Elasticsearch([host], verify_certs=False)
+        es = elasticsearch.Elasticsearch(host if isinstance(host, list) else [host], verify_certs=False)
```

Comment:
Seems like we need to update this based on which engine we're using? If so, we likely should change the "service" field accordingly:

https://github.com/sdskit/sdscli/blob/295bba8bfe5634f49a9f6cf331589770898d3361/sdscli/adapters/hysds/status.py#L78-L79

Reply:
good catch, but luckily elasticsearch's python API is backwards compatible with opensearch (for the most part)
would it be worth changing?

Reply:
Oh nice! In terms of the python API, I think we can leave as-is then. In terms of those print statements (line 78 and 79), I think from an OPS perspective, it would be nice to print out the proper ES engine that we're using. I suppose as an alternative, we could update that term to be more generic like "ElasticSearch Engine" or something to that nature if we didn't want to rely on having to read in the SDS config.